gamestonk_terminal/main_helper.py

```diff
@@ -117,7 +119,8 @@ def load(l_args, s_ticker, s_start, s_interval, df_stock):
     parser = argparse.ArgumentParser(
         add_help=False,
         prog="load",
-        description=""" Load a stock in order to perform analysis.""",
+        description=""" Load a stock in order to perform analysis.
+                        To load an Indian ticker use '.NS' at the end, e.g. 'SBIN.NS'. """,
```

Comment:
Should we specify the indian ticker?  For example, french tickers end in .PA, and I am sure there are many others. (Unless the issue was specifically .NS)

Reply:
Yea, at least 3/4 people requested this. But you gave me a good idea, I'll add this link:
https://help.yahoo.com/kb/exchanges-data-providers-yahoo-finance-sln2310.html